auth/index.html

```diff
@@ -1132,7 +1131,7 @@ <h2>Message log</h2>
         if (event.data && event.data["type"] == "RESET_EMBEDDED_KEY") {
```

Comment:
Note: I've verified that even older versions of `@tkhq/iframe-stamper` never used `RESET_EMBEDDED_KEY`. We can be confident that changing this handler won't affect any existing clients.